.github/workflows/python-tests.yml

```diff
@@ -18,10 +18,9 @@ jobs:
           python-version: '3.10'
           architecture: x64
       - name: Install dependencies
-        # python_tests requires optuna>=3.0.0 since it imports FloatDistribution
         run: |
           python -m pip install --progress-bar off --upgrade pip setuptools
-          pip install --progress-bar off "optuna>=3.0.0"
+          pip install --progress-bar off "optuna>=3.1.0"
```

Comment:
Can we just remove this line since `optuna>=3.1.0` will be installed at the following line?

```suggestion
```